src/presentation/handlers/health_check.rs

```diff
@@ -0,0 +1,23 @@
+#[derive(serde::Serialize, utoipa::ToSchema)]
+pub struct HealthResponse {
+    status: String,
+}
+
+impl axum::response::IntoResponse for HealthResponse {
+    fn into_response(self) -> axum::response::Response {
+        (axum::http::StatusCode::OK, axum::Json(self)).into_response()
+    }
+}
+
+#[utoipa::path(
+    get,
+    path = "/health",
```

Comment:
It is necessary to try to take it as a constant. It will eliminate the problem when the endpoint is updated, but the documentation is not.